cmake-tests/test-install.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -e
+
+SOURCE="${BASH_SOURCE[0]}"
+while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
+  DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+  SOURCE="$(readlink "$SOURCE")"
+  [[ $SOURCE != /* ]] && SOURCE="$DIR/$SOURCE" # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
+done
+SOURCE_DIR="$( cd -P "$( dirname "$SOURCE" )" && cd .. && pwd )"
+
+TMP_DIR=$(mktemp -d)
+
+INSTALL_DIR=$TMP_DIR/hcc
+
+mkdir $TMP_DIR/hcc-build && cd $TMP_DIR/hcc-build
+
+cmake $SOURCE_DIR -DCMAKE_INSTALL_PREFIX=$INSTALL_DIR $@
+make -j`nproc` world
+make -j`nproc`
+make install
+
```

Comment:
Since we install in a non-standard path, the library will need to append the `CMAKE_PREFIX_PATH` with `/opt/rocm` since that is a common location for the runtime. That still won't prevent the user from setting `CMAKE_PREFIX_PATH` themselves when they have installed the runtime in an alternative location.